packages/loader/lib/main.js

```diff
@@ -59,6 +64,8 @@ var mainContext = this;
       for (var i = 0; i < length; i++) {
         if (deps[i] === 'exports') {
           reified[i] = exports;
+        } else if (deps[i] === 'require') {
```

Comment:
Wouldn't it be easier to just `enified('require', function() { return { default: require }; });` instead of changing this?